wolfcrypt/src/rsa.c

```diff
@@ -4419,13 +4436,37 @@ int wc_MakeRsaKey(RsaKey* key, int size, long e, WC_RNG* rng)
 
 #ifdef WOLF_CRYPTO_CB
     if (key->devId != INVALID_DEVID) {
+    #ifndef WOLF_CRYPTO_CB_ONLY_RSA
         err = wc_CryptoCb_MakeRsaKey(key, size, e, rng);
         if (err != CRYPTOCB_UNAVAILABLE)
             goto out;
         /* fall-through when unavailable */
+    #else
+        err = wc_CryptoCb_MakeRsaKey(key, size, e, rng);
```

Comment:
Can you avoid duplicating this call? The file mod changed to 755, please remove execute bit on it.

Reply:
removed duplicating call. remove execution bit. Thanks!